index.js

```diff
@@ -9,13 +9,40 @@ const mongoose = require("mongoose");
 app.use(cors());
 app.use(bodyParser.json());
 
+const databaseConnection = require("./database/dbConnect");
+
+(async () => {
+  await databaseConnection();
+})();
```

Comment:
we can normally call the function here i.e databaseConnection()
No need of async await because in function definition you already defined it as async await